image-fetcher/deployment.yaml

```diff
@@ -18,6 +18,8 @@ spec:
 {% if deploy %}
       priorityClassName: production
 {% endif %}
+      nodeSelector:
+        preemptible: "true"
```

Comment:
This is a daemon, it should be running on all nodes.  It should have a preemptible toleration, but no no selector.